release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -797,21 +798,21 @@ In the table below, features are marked with the following statuses:
 |`oc` CLI Plug-ins
 |TP
 |TP
-|
+|TP
 
 |Service Catalog
-|-
-|-
-|-
+|DEP
+|DEP
+|DEP
 
 |Template Service Broker
-|-
-|-
-|-
+|DEP
+|DEP
+|DEP
 
 |OpenShift Ansible Service Broker
-|-
-|-
+|DEP
+|DEP
 |-
 
```

Comment:
@adellape did I get the Service Catalog, TSB, and ASB statuses right in the TP table? I based those statuses off what I saw in the tracker comments and what we currently have in the RNs.

Reply:
Confirmed via Slack :+1: